### PR TITLE
[15.0] [MIG] product_cost_security

### PR DESCRIPTION
**Pull request by celm1990**

### CODE REVIEW

File: product_cost_security/models/product.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+from odoo import fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    standard_price = fields.Float(groups="product_cost_security.group_product_cost")
+
+
+class ProductProduct(models.Model):
```
Comment (rousseldenis) [inline product_cost_security/models/product.py]: Could you split one model per file ?

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Thanks for your contribution

**Comment by celm1990** [comment]
@rousseldenis is OK???

**Review by rousseldenis — APPROVED** [review]
Could you squash your commits ?

**Comment by celm1990** [comment]
somebody can help me review this PR?
ping @OCA/oca-members  @OCA/product-maintainers

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**